### ISSUE TITLE
Permission with empty slug

### ISSUE THREAD

**Issue by nglelinh** [kodeine/laravel-acl#34]
1) Get a user
2) Add new permission, e.g. update.user
3) Remove permission update.user

Actual: permission "user" with empty slug
Expect: remove whole permission, as we have for role's permission

**Comment by kodeine** [comment]
@nglelinh 
`$user->removePermission('view.user')` will remove the `view.user` permission.
`$user->removePermission('user')` will remove the whole `user` alias permission.

Is it not like that? Last i checked it was working.

**Comment by nglelinh** [comment]
@kodeine: yes, it works as you say. But I mean that, after removing last permission in an alias (empty slug), the whole alias should be automatically deleted as well. We have the same behavior with role permissions as well.
